docs/Makefile

```diff
@@ -17,5 +17,7 @@ help:
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
+	# download the current zenodo bibtex cite
+	curl https://zenodo.org/records/13899727/export/bibtex > source/zenodo.bibtex.txt
```

Comment:
It looks like `13899727` is specifically 2.4.0. Unfortunately, `https://zenodo.org/records/1202434/export/bibtex` redirects to `/records/13899727`, not `/records/13899727/export/bibtex`.

Reply:
oh... let me try again then

Reply:
Here we go:
```suggestion
	curl -L -H 'Accept: application/x-bibtex' https://zenodo.org/api/records/1202434 > source/zenodo.bibtex.txt
```

Reply:
nice.  this is much harder to do than it should be